File: problema01/sol_java/problema01/src/problema01/Problema01.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package problema01;

import java.util.Locale;
import java.util.Scanner;

/**
 *
 * @author PabloCosta14
 */
public class Problema01 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        Scanner entrada = new Scanner(System.in);
        entrada.useLocale(Locale.US);

        double anguloA;
        double anguloB;
        double anguloC;
        String tipo_triangulo = "";

        System.out.println("ingrese el valor del angulo A");
        anguloA = entrada.nextDouble();

        System.out.println("ingrese el valor del angulo B");
        anguloB = entrada.nextDouble();

        System.out.println("ingrese el valor del angulo C");
        anguloC = entrada.nextDouble();

        if (anguloA == 90 || anguloB == 90 || anguloC == 90) {
            tipo_triangulo = "rectangulo";
        } else {
            if (((anguloA > (90)) && (anguloA <= (180)))
                    || ((anguloB > (90)) && (anguloB <= (180)))
                    || ((anguloC > (90)) && (anguloC <= (180)))) {
                tipo_triangulo = "obtusangulo";
            } else {
                if (anguloA < 90 || anguloB < 90 || anguloC < 90) {
                    tipo_triangulo = "Acutangulo";
                }
            }
        }
         System.out.printf("el triangulo es tipo:\n" + "%s\n",
                tipo_triangulo);

    }
}       

File: problema01/sol_python/problema01.py


anguloA = float(input("Ingrese el valor del angulo A\n"))
anguloB = float(input("Ingrese el valor del angulo B\n"))
anguloC = float(input("Ingrese el valor del angulo C\n"))

if anguloA == 90 or anguloB == 90 or anguloC == 90:
    tipo_triangulo = "Rectangulo"
else:
    if anguloA > 90 and anguloA <= 180 or anguloB > 90 and anguloB <= 180 or anguloC > 90 and anguloC <= 180:
        tipo_triangulo = "Obtusangulo"
    else:
        if anguloA < 90 or anguloB < 90  or  anguloC < 90:
            tipo_triangulo = "Acutangulo"

print("el triangulo es de tipo\n",tipo_triangulo )